lightly/api/api_workflow_export.py

```diff
@@ -262,40 +273,50 @@ def export_filenames_and_read_urls_by_tag_id(
             tag_id=tag_id,
             file_name_format=FileNameFormat.REDIRECTED_READ_URL,
         )
+        datasource_urls_string = retry(
+            self._tags_api.export_tag_to_basic_filenames,
+            dataset_id=self.dataset_id,
+            tag_id=tag_id,
+            file_name_format=FileNameFormat.DATASOURCE_FULL,
+        )
```

Comment:
What happens for this endpoint with old datasets that were created using the local workflow? My fear is that an empty string is returned which will result in `zip(filenames, read_urls, datasource_urls)` being and empty list.

Reply:
It works, I tried it out for a dataset uploaded via CLI:
![image](https://user-images.githubusercontent.com/20324507/231413932-d564c25e-3607-4d47-abff-043570f50b9a.jpeg)

Reply:
Awesome!